client/my-sites/activity/filterbar/style.scss

```diff
@@ -172,26 +175,29 @@
 		}
 	}
 
-	.DayPicker-Day--start {
+	.DayPicker-Day.DayPicker-Day--start {
 		background-color: var( --color-primary-light );
 		border-radius: 0;
 		border-top-left-radius: 50%;
 		border-bottom-left-radius: 50%;
 	}
 
-	.DayPicker-Day--end {
+	.DayPicker-Day.DayPicker-Day--end {
```

Comment:
There's a specificity war going on with the DatePicker IMHO.

I believe we need to fundementally strip back the range styles and put them into the core DatePicker `styles.scss` file leaving only the truely _custom_ styling exceptions in the core `DateRange` and this Activity date range.